src/components/course-periods-nav.cjsx

```diff
@@ -57,7 +58,14 @@ CoursePeriodsNav = React.createClass
     @setState(active: key)
 
   renderPeriod: (period, key) ->
-    <BS.NavItem eventKey={key} key="period-nav-#{period.id}">{period.name}</BS.NavItem>
+    {plan} = @props
+    isTrouble = _.pluck(plan.stats, 'is_trouble')
```

Comment:
It looks like this puts a trouble flag on all periods if any contain `is_trouble`. I do not think this is the desired behavior.